doc/develop/modules.rst

```diff
@@ -1031,7 +1031,9 @@ module.
 To avoid merging changes to master with pull request information, the pull
 request should be marked as ``DNM`` (Do Not Merge) or preferably a draft pull
 request to make sure it is not merged by mistake and to allow for the module to
-be merged first and be assigned a permanent commit hash. Once the module is
+be merged first and be assigned a permanent commit hash. Drafts reduce noise by
+not automatically notifying anyone until marked as "Ready for review".
+Once the module is
```

Comment:
minor nit.
Very short line, maybe the line below should be joined with this line, although I acknowledge it will lead to more line changes as the line then must be wrapped and one more line to join.

Reply:
Yes that's why I left it like this: to avoid a much larger diff. This has of course no effect on the final HTML output.